Tests/Tests/CDTQIndexManagerTests.m

```diff
@@ -202,4 +203,58 @@
 
     });
 
+    describe(@"when managing search limitations", ^{
```

Comment:
There wasn't a clear place to put this test.  I believe that it belongs in the index manager tests and I want to have a somewhat generic `describe` and then making it more specific in the `it` block.  Anyway, I changed the `describe` to be more specific in 288ac97.